scripts/flamingfir-deploy.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+TIMEOUT=60
+TARGET_HOST="$1"
+COMMIT=$(echo ${CI_BUILD_REF} | cut -c -9)
+DOWNLOAD_URL="https://releases.parity.io/substrate/x86_64-debian:stretch/2.0.0-${COMMIT}/substrate"
+POST_DATA='{"extra_vars":{"artifact_path":"'${DOWNLOAD_URL}'","target_host":"'${TARGET_HOST}'"}}'
+
+JOB_ID=$(wget -O - --header "Authorization: Bearer ${AWX_TOKEN}" --header "Content-type: application/json" --post-data "${POST_DATA}" https://ansible-awx.parity.io/api/v2/job_templates/32/launch/ | jq .job)
+
+echo "Launched job: $JOB_ID"
+
+sleep $TIMEOUT
+
+AWX_OUTPUT=$(wget -O - --header "Authorization: Bearer ${AWX_TOKEN}"  https://ansible-awx.parity.io/api/v2/jobs/${JOB_ID}/stdout?format=txt_download)
+
```

Comment:
I admin that it can be considered over-engineering :eyeglasses: , and I think it's very good like that but I see the danger of a timeout-caused misinformation here. When iterating one could reduce the time to wait initially and at one point the job would definitely have a binary status on awx.

Reply:
@gabreal good point! So now it will do (as default) 10 attempts each 15 seconds which can be configured by the script. So it does work faster and can be updated later. See the latest commit.